lib/utils/helper_function.js

```diff
@@ -1,14 +1,19 @@
 const { OK_CODE, NOT_FOUND_CODE } = require('../../constants/');
 
+const defaultResponse = (res, message, statusCode = OK_CODE) => res.status(statusCode).json({
```

Comment:
What is this function `defaultResponse` solving for?

Reply:
successResponse method was created with four arguments and since I am not returning any payload (data), I was advised to create a new one called defaultResponse.

Reply:
While transmitting objects from server to client or vice versa undefined object attribute are removed from a js object. I believe objects are stringified before been sent over the wire thus, the removal of undefined attributes from an object. Try running the code snippet on the console to get a feel of what I am saying.

```js
JSON.stringify({data:undefined, name:"me"})  // output  = "{"name":"me"}"
```

Reply:
Fixed